### PR TITLE
Correct heron developers doc

### PR DESCRIPTION
**Pull request by maosongfu**
1. Correct out-dated docs.
2. Fix broken links.

### CODE REVIEW

File: website/content/docs/contributors/custom-scheduler.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ title: Implementing a Custom Scheduler
 ---
 
 To run a Heron cluster, you'll need to set up a scheduler that is responsible
-for cluster management. Heron currently supports two schedulers out of the box:
+for cluster management. Heron currently supports the followings schedulers out of the box:
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/followings/following

File: website/content/docs/contributors/custom-scheduler.md [hunk 2]
```diff
@@ -47,48 +47,28 @@ interfaces:
...
-[`IConfigLoader`](/api/com/twitter/heron/spi/scheduler/IConfigLoader.html).
-
-## Trying Out Your Scheduler
-
-Once you've implemented a custom configuration loader, you'll need to specify
-your loader by class using the `--config-loader` flag. If your loader relies on
-a configuration file, specify the path of that file using the `--config-file`
-flag. Here's an example [topology
-submission](../../operators/heron-cli#submitting-a-topology) command:
+To run the a custom scheduler, the implementation of the interface above must be specified in the [config](../../operators/deployment/configuration) .
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: no space before '.'

File: website/content/docs/contributors/custom-scheduler.md [hunk 3]
```diff
@@ -47,48 +47,28 @@ interfaces:
...
+By default, the heron-cli looks for config under `${HERON_HOME}/conf/`. The location can be overridden using option `--config-path`.
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/config/configurations

File: website/content/docs/contributors/custom-scheduler.md (hunk 1 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: done.

File: website/content/docs/contributors/custom-scheduler.md (hunk 2 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: done

File: website/content/docs/contributors/custom-scheduler.md (hunk 3 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: done

File: website/content/docs/contributors/custom-scheduler.md [hunk 4]
```diff
@@ -47,48 +47,28 @@ interfaces:
...
+$ heron submit [cluster-name-storing-your-new-config]/[role]/[env] \
+    --config-path [config-folder-path-storing-your-new-config] \
     /path/to/topology/my-topology.jar \
-    biz.acme.topologies.MyTopology \
-    --config-file=/path/to/config/my_scheduler.conf \
-    --config-loader=biz.acme.config.MyConfigLoader
+    biz.acme.topologies.MyTopology 
 ```
+
+The implementation for each of the interface listed above must be on Heron's
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/interface/interfaces

**Comment by billonahill** [comment]
One last comment. Fix it and ship it.

👍

**Pull request merged**